sensor/tests/replay/replay_test.go

```diff
@@ -293,6 +323,23 @@ func readSensorOutputFile(fname string) ([]*central.MsgFromSensor, error) {
 	return msgs, nil
 }
 
+func getAlerts(messages []*central.MsgFromSensor) map[string]map[string]*storage.Alert {
+	events := make(map[string]map[string]*storage.Alert)
+	for _, msg := range messages {
+		event := msg.GetEvent()
+		if event.GetAlertResults() != nil {
+			if event.GetAlertResults().GetDeploymentId() != "" {
+				alerts := make(map[string]*storage.Alert)
```

Comment:
Maybe add the map size if you know it ahead of time:
```golang
alertResults := event.GetAlertResults().GetAlerts()
alerts := make(map[string]*storage.Alert, len(alertResults))
for _, a := range alertResults {
       alerts[a.GetPolicy().GetId()] = a
}
```

Reply:
👍